<launch>
   <arg name="motor_num" default="0" />
   <arg name="set_point" default="40" />
   <node name="motor_$(arg motor_num)_pid" pkg="pid" type="controller" output="screen" >
      <param name="Kp" value="0.001" />
      <param name="Ki" value="0" />
      <param name="Kd" value="0" />
      <param name="upper_limit" value="1" />
      <param name="lower_limit" value="-1" />
      <param name="windup_limit" value="10" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="100.0" />
      <remap from="/setpoint" to="/motor$(arg motor_num)/setpoint" />
      <remap from="/state" to="/motor$(arg motor_num)/state" />
      <remap from="/control_effort" to="/motor$(arg motor_num)/control_effort" />
      <remap from="/pid_enable" to="/motor$(arg motor_num)/pid_enable" />
   </node> 

    <node name="motor$(arg motor_num)_effort" pkg="my_wro" type="control_effort.py" output="screen" >
	    <param name="motor_num" value="$(arg motor_num)" />
	    <param name="set_point" value="$(arg set_point)" />
	</node>
</launch>